samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages_eu.properties

```diff
@@ -577,3 +577,4 @@ seb_download_configuration=Deskargatu konfigurazioa
 seb_download_seb=Deskargatu Safe Exam Browser
 seb_launch_seb=Exekutatu Safe Exam Browser
 seb_quit_browser=Atera nabigatzaile segurutik (Safe Exam Browser)
+seb_preview_info=Assessments that are configured to use Safe Exam Browser can not be previewed.
```

Comment:
Hi @PabloSanRoman, could you help with this translation please?

Reply:
Hi @stetsche !
seb_preview_info=Ezin dira aurreikusi Safe Exam Browserako konfiguratutako azterketak.

Reply:
Thank you!